docs/manage-your-account/workspace-usage.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 3
+title: Workspace Usage
+description: "Understand your workspace usage"
+---
+
+import orgUsage from "/static/ui-videos/usage/usage-fhd.mp4";
+
+Keeping track of your workspace usage is essential to ensure that your team is operating at peak efficiency and that your
+organization is getting the most out of Permit.
+
+## Workspace Usage
+
+To view your workspace usage, go to the **Projects** screen on the Sidebar.
+The workspace usage is displayed in the top right corner of the screen.
+In addition to the current usage, clicking on the (?) icon will show you the usage history of your workspace tenants and MAUs.
+
+<video controls autoPlay loop muted playsInline>
```

Comment:
One small thing with the recordings: 
- I want to make sure we have a consistent way out UI recordings look (and this is something I need to change too) but lets remove all padding and rounded corners from screen.studio settings. ;)

Reply:
Done :)